drivers/src/pci/config_space.rs

```diff
@@ -5,7 +5,7 @@
 use core::ops::Range;
```

Comment:
Nit: In commit message "it's location" should be "its location".

Reply:
Done.